httpmw/operationid/operationid.go

```diff
@@ -0,0 +1,44 @@
+package operationid
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+type contextKey string
+
+const (
+	OperationIDKey   contextKey = "operationID"
+	CorrelationIDKey contextKey = "correlationID"
+	// http header name ACS RP adds for it's operation ID
+	RequestAcsOperationIDHeader = "x-ms-acs-operation-id"
```

Comment:
From the name, this seems to be AKS' own header. If so, please comment it explicitly.

Reply:
added headers from toolkit log as default

Reply:
Removed.